pallets/parachain-staking/src/lib.rs

```diff
@@ -100,6 +100,7 @@ pub mod pallet {
 	#[pallet::without_storage_info]
 	pub struct Pallet<T>(PhantomData<T>);
 
+	pub type Slot = u64;
```

Comment:
Please don't redefine a Slot type, use the existing one from `sp_consensus_slots`: https://paritytech.github.io/polkadot-sdk/master/sp_consensus_slots/struct.Slot.html

Reply:
I didn't use the `Slot` type from `sp_consensus_slots` because Moonbeam and Moonriver will still use the para-block number and is easy to convert its type starting from u64, but it's true that in the future we will adapt all the chains to use the `Slot` type coming from the relay, so I will update it now to prepare the pallet for the future.